vumi/servicemaker.py

```diff
@@ -197,6 +203,12 @@ class VumiWorkerServiceMaker(object):
     # what command line options does this service expose
     options = StartWorkerOptions
 
+    def set_maxthreads(self, options):
```

Comment:
Why are we passing in the options object here instead of the `maxthreads` value?